training/train.py

```diff
@@ -58,22 +63,23 @@ def criteria_satisfied(current_loss, _):
 model = ModelBuilder(alpha=0.25).to(device)
 
 parameters = list(model.parameters())
-optimizer = torch.optim.Adam(parameters, lr=lr)
+optimizer = torch.optim.Adam(parameters, lr=0.05)
```

Comment:
Move lr into variable

Reply:
@macoidk Please don't change this file in this PR. If you suggest improvements to it, make a separate PR.